setup.cfg

```diff
@@ -1,10 +1,219 @@
+    # * initializer, needed by logic in our custom `Request` subclass.
+    #
+    #   In a perfect world, Twisted[conch] would be a dependency of an sftp
+    #   extra.  However, pip fails to resolve the dependencies all
+    #   dependencies when asked for Twisted[tls] *and* Twisted[conch].
+    #   Specifically, Twisted[conch] (as the later requirement) is ignored.
+    #   If there were an Tahoe-LAFS sftp extra that dependended on
+    #   Twisted[conch] and install_requires only included Twisted[tls] then
+    #   `pip install tahoe-lafs[sftp]` would not install requirements
+    #   specified by Twisted[conch].  Since this would be the *whole point* of
+    #   an sftp extra in Tahoe-LAFS, there is no point in having one.
+    # * Twisted 19.10 introduces Site.getContentFile which we use to get
+    #   temporary upload files placed into a per-node temporary directory.
+    Twisted[tls,conch] >= 19.10.0
+
+    PyYAML >= 3.11
+
+    six >= 1.10.0
+
+    # for 'tahoe invite' and 'tahoe join'
+    magic-wormhole >= 0.10.2
+
+    # A new enough version to support custom JSON encoders.
+    eliot >= 1.13.0
+
+    # Pyrsistent 0.17.0 (which we use by way of Eliot) has dropped
+    # Python 2 entirely; stick to the version known to work for us.
+    pyrsistent
+
+    # A great way to define types of values.
+    attrs >= 18.2.0
+
+    # WebSocket library for twisted and asyncio
+    autobahn >= 19.5.2
+
+    # Support for Python 3 transition
+    future >= 0.18.2
+
+    # Discover local network configuration
+    netifaces
+
+    # Utility code:
+    pyutil >= 3.3.0
+
+    # Linux distribution detection:
+    distro >= 1.4.0
+
+    # For the RangeMap datastructure. Need 2.0.2 at least for bugfixes.
+    collections-extended >= 2.0.2
+
+    # Duplicate the Twisted pywin32 dependency here.  See
+    # https://tahoe-lafs.org/trac/tahoe-lafs/ticket/2392 for some
+    # discussion.
+    pywin32 != 226 ; sys_platform=="win32"
+
+    # HTTP server and client
+    klein
+
+    # 2.2.0 has a bug: https://github.com/pallets/werkzeug/issues/2465
+    werkzeug != 2.2.0
+    treq
+    cbor2
+    pycddl >= 0.2
+
+    # for pid-file support
+    psutil
+    filelock
+
+[options.packages.find]
+# inform the setuptools source discovery logic to start in this directory
+where = src
+
+[options.package_data]
+allmydata.web =
+    *.xhtml
+    static/*.js
+    static/*.png
+    static/*.css
+    static/img/*.png
+    static/css/*.css
+
+[options.extras_require]
+build =
+      dulwich
+      gpg
+
+test =
+    flake8
+    # On Python 3.7, importlib_metadata v5 breaks flake8.
+    # https://github.com/python/importlib_metadata/issues/407
+    importlib_metadata < 5; python_version < "3.8"
+
+# Pin a specific pyflakes so we don't have different folks
+    # disagreeing on what is or is not a lint issue.  We can bump
+    # this version from time to time, but we will do it
+    # intentionally.
+    pyflakes == 2.2.0
```

Comment:
This indenting seems weird (at least _all_ of the comment should be indented the same?)